pages/game_page.py

```diff
@@ -1,3 +1,33 @@
 import streamlit as st
+import random
 
-st.title("GAME PAGE")
\ No newline at end of file
+st.title("Build a Scientist")
+
+game_words = ['Genetic','Histology','Sequence','Clinical Trial','Mutation','Genome','Oncology']
+
+def game_word_generator(word): 
+    st.write(word)
+    st.write(f"`{' '.join(['_'] * len(word))}`")
+    
+if game_words:
+    selected_word = random.choice(game_words)
+    game_word_generator(selected_word)
+   
+# Session state
+if 'selected_word' not in st.session_state:
+    st.session_state.selected_word = game_word_generator(selected_word)
```

Comment:
This sets `st.session_state.selected_word` to the return value of the `game_word_generator` but that function doesn't return anything. Instead you need
```suggestion
    st.session_state.selected_word = selected_word
```

Reply:
Omg this makes way more sense, idk what I was doing. Thank you!